supplier/tests/test_supplier_sign_up.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from django.template.loader import get_template
+from supplier.models import Supplier
+
+
+@pytest.mark.django_db()
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template('supplier/supplier_sign_up.html')
+
+    def test_signup_entry_point(self, client):
+        response = client.get('/supplier/signup')
+        assert response.status_code == 200
+
+    def test_signup_user(self, client):
+        response = client.post(
+            '/supplier/signup', {'user_name': "testSignUp",
+                                 'password': "pokemon123",
+                                 'first_name': "ash",
+                                 'last_name': "katchamp",
+                                 'business_name': "pokemon inc"}
+        )
+        assert Supplier.objects.get(user_name="testSignUp") in list(Supplier.objects.all())
```

Comment:
shouldn't we expect an encrypted password?

Reply:
Done!